### PR TITLE
Add streamable.com video plugin

### PR DESCRIPTION
**Pull request by JCallicoat**
Support inline previews for streamable.com videos.

Example: https://streamable.com/xh1q1

### CODE REVIEW

**Comment by lorenzhs** [comment]
Hey @JCallicoat, thanks for sending this PR. Is there a particular reason why you're not using `UrlPlugin`? It makes handling URLs a bit easier and handles multiple occurrences, too (so you can have two videos linked in a message).

File: js/plugins.js [hunk 1]
```diff
@@ -539,8 +539,29 @@ plugins.factory('userPlugins', function() {
         }
     });
 
+    /*
+     * Streamable Embedded Player
+     */
+    var streamablePlugin = new Plugin('Streamable video', function(message) {
+        var rPath = /streamable\.com\/s?\/?(.+)/;
```
Comment (lorenzhs) [inline js/plugins.js]: this might match unrelated stuff with `streamable.com/blah` anywhere in a URL, it would probably be best to include the protocol explicitly.

**Comment by JCallicoat** [comment]
@lorenzhs I hadn't really looked at UrlPlugin, I will have a go at making the plugin use it!

**Comment by JCallicoat** [comment]
Switched to using `UrlPlugin` and rebased.

**Comment by lorenzhs** [comment]
Thanks!
